ui/src/views/storage/FileShareTab.vue

```diff
@@ -0,0 +1,202 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+<template>
+  <a-spin :spinning="loading">
+    <a-tabs
+      :activeKey="currentTab"
+      :tabPosition="device === 'mobile' ? 'top' : 'left'"
+      :animated="false"
+      @change="handleChangeTab">
+      <a-tab-pane :tab="$t('label.details')" key="details">
+        <DetailsTab :resource="dataResource" :loading="loading" />
+      </a-tab-pane>
+      <a-tab-pane :tab="$t('label.access')" key="access">
+        <h3>{{ $t('label.mount.fileshare') }}</h3>
+        <div v-for="(nic, index) in resource.nic" :key="index" class="content">
+          <a-card>
+            <template #title>
+              <router-link :to="{ path: '/guestnetwork/' + nic.networkid }">
+                {{ nic.networkname }}
+              </router-link>
+            </template>
+            <timeline>
+              <a-timeline-item color="blue">
+                <h3>mount -t nfs -o rw,sync {{ nic.ipaddress }}:{{ resource.path }} [local_mount_path]</h3>
```

Comment:
would be great to get a copy button for commands but it is just UX and can be added later

Reply:
I tried but couldn't find out how to do it. Will take your help on this.